src/recognizer/recognizer.py

```diff
@@ -0,0 +1,88 @@
+import time
+import tensorflow_hub as hub
+import numpy as np
+from matplotlib import pyplot as plt
+from PIL import Image
+
+
+class Recognizer():
+
+    def display_image(self, image):
+        """Displays an image"""
+        fig = plt.figure(figsize=(20, 15))
+        plt.grid(False)
+        plt.imshow(image)
+        plt.show();
+
+    def test_display_image(self):
+        print("Testing image show from PLT")
+        image = Image.open("../../tests/data/images/test_scene.jpeg")
+        self.display_image(image)
+
+    def draw_bounding_box_on_image(self, image, ymin, xmin, ymax, xmax,
+                                     color, font, thickness=4, display_str_list=()):
```

Comment:
Should color and/or font also have a default value?

Reply:
`draw_bounding_box_on_image()` is only called from `draw_boxes()` which hashes the classes to a color and loads a default font.